lib/galaxy/webapps/galaxy/api/workflows.py

```diff
@@ -65,6 +65,7 @@ def index(self, trans, **kwd):
             encoded_id = trans.security.encode_id(wf.id)
             item['url'] = url_for('workflow', id=encoded_id)
             item['owner'] = wf.user.username
+            item['latest_workflow_steps'] = len( wf.latest_workflow.steps )
```

Comment:
`latest_workflow_steps` was a little confusing at first, maybe `step_number` or `number_of_steps` would be clearer?